### DISCUSSION TITLE
Stream the bbox coordinates to another running program

### DISCUSSION THREAD

**Discussion by fauzan-ragitya** [AlexeyAB/darknet#7603] [Q&A]
Hi, I currently build a project with darknet and tiny YOLOV4 on Single Board Computer (Jetson Nano) to detect a person and do visual servoing. I have done the detection part and able to stream a real time video with bounding box to locate the person in the image. But, I have a problem to do the visual servoing part. Is there a way to modify the repository and add my own visual servoing script accessing the bounding box coordinates? Or can I stream the coordinates to another running program outside the repo?

**Comment by ljbkusters** [comment]
Hi fauzan,

I'm working on a somwhat similar project. I was thinking the coordinate stream could be achieved using sockets, though I have not yet had the time to implement this. Right now these coordinates are being written to a txt file in our case, but it should be possible to add a sockets channel (for me python server (script) <-> darknet). 

However I've been having issues finding how to implement a video stream into the project. Do you have any idea where to look? The "video-camera mjpeg-stream with any Android smartphone" guide is not applicable to my case.

Cheers
